modules/KoreBuild.Tasks/CodeSign.targets

```diff
@@ -52,7 +30,7 @@
       DryRun="$(SignToolDryRun)"
       TestSign="$(SignToolTestSign)"
       ItemsToSign="@(_ItemsToSign)"
-      FileSignInfo="@(_FileSignInfo)"
+      FileSignInfo="@(_FileSignInfo);@(FileNamesToSign)"
```

Comment:
Where are `@(FileNamesToSign)` items created and why aren't they included in `@(_FileSignInfo)`?

Reply:
They are optionally specified by a repo. I'll be using this item group when I cleanup these files: https://github.com/aspnet/AspNetCore/blob/master/build/CodeSign.props, https://github.com/aspnet/AspNetCore/blob/master/build/CodeSign.targets

Reply:
Yup, that helped